stories/legacy.stories.js

```diff
@@ -1,6 +1,6 @@
 import { setAuthTokenWithOAuthCredentials } from './storiesUtils';
 
-import { legacyGetMapFromUrl, ApiType } from '../dist/sentinelHub.esm';
+import { legacyGetMapFromUrl, ApiType, BBox, CRS_EPSG3857, legacyGetMapFromParams } from '../dist/sentinelHub.esm';
```

Comment:
`BBox, CRS_EPSG3857` seem to not be used in here.

Reply:
Thanks, fixed.

Reply:
Resolving.